cmd/createtree/main_test.go

```diff
@@ -138,18 +138,16 @@ func TestRun(t *testing.T) {
 		},
 	}
 
+	ctx := context.Background()
 	for _, test := range tests {
 		test := test
 		func() {
 			var out *string
-			println = func(a ...interface{}) (int, error) {
+			printer := func(a ...interface{}) (int, error) {
 				str := fmt.Sprintln(a...)
 				out = &str
 				return 0, nil
 			}
-			defer func() {
-				println = fmt.Println
-			}()
 
 			if test.createErr != nil {
```

Comment:
Why not just set `server.err = test.CreateErr` at the start of each loop iteration - no `if`, no `defer` ?